### DISCUSSION TITLE
Welcome to amidon Discussions!

### DISCUSSION THREAD

**Discussion by midwan** [BlitterStudio/amidon#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lemart1** [comment]
Not working with OS 3.2.2.1 & Texteditor.mmc version 15.56... Error you need version 15.55!
If you made it with Hollywood (hwa file) you should tell the people which H-Player  are they using...
![grafik](https://user-images.githubusercontent.com/132704685/236546512-1c3920b0-9766-452e-b0f3-9a41ec06d47d.png)

**Reply by midwan** [reply]
I have the same OS version here, and TextEditor.mcc 15.56, and I'm not seeing any errors. Both on MUI 5 and 3.9...
